### PR TITLE
Enable sync cursor for multifile project when there is noly one preview server

### PR DESCRIPTION
**Pull request by 7sDream**
As discussion in #164, this is a incomplete, but simple and acceptable temporary solution for multifile project.

### CODE REVIEW

File: addons/vscode/src/extension.ts [hunk 1]
```diff
@@ -95,9 +95,13 @@ const serverProcesses: Array<any> = [];
 const activeTask = new Map<vscode.TextDocument, TaskControlBlock>();
 
 const reportPosition = async (bindDocument: vscode.TextDocument, activeEditor: vscode.TextEditor, event: string) => {
-	const tcb = activeTask.get(bindDocument);
+	let tcb = activeTask.get(bindDocument);
 	if (tcb === undefined) {
-		return;
+		if (activeTask.size === 1) {
```
Comment (Enter-tainer) [inline addons/vscode/src/extension.ts:100]: can you add some comment about the hack here?

File: addons/vscode/src/extension.ts [hunk 2]
```diff
@@ -276,16 +280,19 @@ const launchPreview = async (task: LaunchInBrowserTask | LaunchInWebViewTask) =>
 	}
 
 	const src2docHandler = (e: vscode.TextEditorSelectionChangeEvent) => {
-		if (e.textEditor === activeEditor) {
+		if (e.textEditor === activeEditor || activeTask.size === 1) {
```
Comment (Enter-tainer) [inline addons/vscode/src/extension.ts:283]: same here

**Review by Enter-tainer — APPROVED** [review]
cool

**Comment by 7sDream** [comment]
Ccomments added.

The `Allow edits by maintainers` option of this PR is enabled, feel free to do changes you want before merge.

**Review by Enter-tainer — APPROVED** [review]
thx

**Pull request merged**